agent/modules/fixer.py

```
@@ -0,0 +1,13 @@
+"""Simple code fixer used by main.py."""
+
+
+def fix_code(raw_code: str) -> str:
+    """Return a "fixed" version of *raw_code*.
+
+    The current implementation is intentionally minimal – it merely returns
```

Ruff (0.13.3)

Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)